src/conversion/view-selection-to-model-converters.js

```diff
@@ -39,10 +39,8 @@ export function convertSelectionChange( modelDocument, mapper ) {
 
 		modelSelection.setRanges( ranges, viewSelection.isBackward );
 
-		if ( !modelSelection.isEqual( modelDocument.selection ) ) {
```

Comment:
I saw the comment in the test. But... is setting the model selection to identical selection triggering the entire conversion (i.e. firing the change events)?  What's the implication of this change to potential consumers of these events? 

In fact, I'd say that the selection should not fire any events if selections are identical. I even think we worked on this, but maybe it was in a different class (maybe it was something about attributes). Anyway, this sounds wrong.

cc @pjasiun